pulp_2to3_migration/app/plugin/content.py

```diff
@@ -246,16 +250,22 @@ def get_remote_by_importer_id(importer_id):
                 select_extra.append('pulp2content__pulp2_repo')
 
             pulp_2to3_detail_qs = pulp_2to3_detail_qs.select_related(*select_extra)
-            for pulp_2to3_detail_content in pulp_2to3_detail_qs.iterator(chunk_size=800):
+            async for pulp_2to3_detail_content in sync_to_async_iterable(
+                    pulp_2to3_detail_qs.iterator(chunk_size=800)
+            ):
                 dc = None
-                pulp2content = pulp_2to3_detail_content.pulp2content
+                pulp2content = await sync_to_async(Pulp2Content.objects.get)(
+                    pulp_2to3_detail_content.pulp2content_id
```

Comment:
I think it needs `pk=`, my bad

Reply:
yeah, I just got to it and figured that, we are perfectly in sync :)